rocminstall.py

```diff
@@ -577,7 +584,8 @@ def workaround_dummy_versionfile_rpm(args, rocmbaseurl):
         or args.revstring[0] == "5.1.1" or args.revstring[0] == "5.1.2" or args.revstring[0] == "5.1.3"
         or args.revstring[0] == "5.2.1" or args.revstring[0] == "5.2.2" or args.revstring[0] == "5.2.3"
         or args.revstring[0] == "5.3.1" or args.revstring[0] == "5.3.2" or args.revstring[0] == "5.3.3"
-        or args.revstring[0] == "5.4.1" or args.revstring[0] == "5.4.2" or args.revstring[0] == "5.4.3"):
+        or args.revstring[0] == "5.4.1" or args.revstring[0] == "5.4.2" or args.revstring[0] == "5.4.3"
+        or args.revstring[0] == "5.5"):
```

Comment:
@sanjtrip  please remove check for `5.5`. The major versions are handled automatically. Only the dot releases need special checks.